data-and-functions-1/src/functions/getMostExpensiveProduct.js

```diff
@@ -0,0 +1,17 @@
+import DATA from '../DATA'
+
+const getMostExpensiveProduct = (data) => {
+  if (data == null) {
+    return null
+  }
+  let mostExpensiveProduct = 0
+  DATA.products.forEach((m) => {
```

Comment:
The `forEach` loop passes in the actual object as the first param to your function. Instead of calling it `m`, call it `currentProduct`